stripe-patrons-data/src/main/scala/com/gu/patrons/lambdas/ProcessStripeSubscriptionsLambda.scala

```diff
@@ -22,29 +22,35 @@ import scala.concurrent.duration.{Duration, DurationInt, MILLISECONDS}
 
 class ProcessStripeSubscriptionsLambda extends RequestHandler[Unit, Unit] {
 
+  private val stage = StageConstructors.fromEnvironment
+  private val stripeConfig = PatronsStripeConfig.fromParameterStoreSync(stage)
+  private val identityConfig = PatronsIdentityConfig.fromParameterStoreSync(stage)
```

Comment:
By loading config synchronously in the class here we cause it to happen during lambda initialisation (at deploy time), rather than during invocation. This reduces invocation duration by about 8 seconds